__e2e__/config.test.ts

```diff
@@ -0,0 +1,45 @@
+import path from 'path';
+import {
+  runCLI,
+  getTempDirectory,
+  cleanup,
+  writeFiles,
+  spawnScript,
+} from '../jest/helpers';
+
+const cwd = getTempDirectory('test_root');
+
+beforeAll(() => {
+  // Register all packages to be linked
+  for (const pkg of ['platform-ios', 'platform-android']) {
+    spawnScript('yarn', ['link'], {
+      cwd: path.join(__dirname, `../packages/${pkg}`),
+    });
+  }
+
+  // Clean up folder and re-create a new project
+  cleanup(cwd);
+  writeFiles(cwd, {});
```

Comment:
cleanup should be sufficient, can we remove writeFiles?

Reply:
We can't because writeFiles function creates folder at cwd path :)